Stage I/Stage I.py

```diff
@@ -40,16 +53,25 @@
 historical2 = historical2.drop(historical2.columns[4:],axis = 1)
 historical2 = historical2.drop(historical2.columns[0],axis = 1)
 
+# Here I would recomment splitting these operations over multiple lines using
+# the &= operator, so something like
+# a = b & c
+# a &= d
+# a &= e
+# instead of a = b & c & d & e
```

Comment:
Yes, you are right. I read the lines incorrectly. I would try something like this:

```
year1under19 = (historical2.YEAR ==2008) & (historical2.AGE <=19)
year2under19 = (historical2.YEAR ==2009) & (historical2.AGE <=19)
pop_dep2 = []
pop_dep2.append(historical2.TOT_POP[year1under19].sum())
pop_dep2.append(historical2.TOT_POP[year2under19].sum())
```